sitestats-impl/src/bundle/site-stats-components.xml

```diff
@@ -345,6 +344,29 @@
         </property>
     </bean>
 
+	<bean id="org.sakaiproject.sitestats.impl.MBeanExporter"
+		  class="org.springframework.jmx.export.MBeanExporter" lazy-init="false">
+		<property name="beans">
+			<map>
+				<entry key="org.sakaiproject.sitestats.api:name=StatsUpdateManager"
+					   value-ref="org.sakaiproject.sitestats.api.StatsUpdateManager"/>
+			</map>
+		</property>
+		<!-- This doesn't work because it detects both the original bean and it's Transaction Proxy -->
+		<!--
+		<property name="autodetectModeName" value="AUTODETECT_MBEAN"/>
+		<property name="namingStrategy">
+			<bean class="org.springframework.jmx.export.naming.KeyNamingStrategy">
+				<property name="mappings">
+					<props>
+						<prop key="org.sakaiproject.sitestats.api.StatsUpdateManager">org.sakaiproject.sitestats.api:name=StatsUpdateManager</prop>
+					</props>
+				</property>
+			</bean>
+		</property>
+		 -->
+	</bean>
+

```

Comment:
Is it useful somehow adding commented out code? Might we need it later or something?